airbyte-integrations/bases/source-acceptance-test/setup.py

```diff
@@ -13,7 +13,7 @@
     "inflection~=0.5",
     "pdbpp~=0.10",
     "pydantic~=1.6",
-    "pytest~=6.1",
+    "pytest~=6.2,>=6.2.4",
```

Comment:
This was because on 6.1, ran into 
```
(.venv) ella@ellarohhflf6qwt source-acceptance-test % python -m pytest unit_tests/test_json_schema_helper.py
Traceback (most recent call last):
  File "/opt/homebrew/Cellar/python@3.10/3.10.8/Frameworks/Python.framework/Versions/3.10/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/opt/homebrew/Cellar/python@3.10/3.10.8/Frameworks/Python.framework/Versions/3.10/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/Users/ella/airbytehq/airbyte/airbyte-integrations/bases/source-acceptance-test/.venv/lib/python3.10/site-packages/pytest/__main__.py", line 5, in <module>
    raise SystemExit(pytest.console_main())
  File "/Users/ella/airbytehq/airbyte/airbyte-integrations/bases/source-acceptance-test/.venv/lib/python3.10/site-packages/_pytest/config/__init__.py", line 187, in console_main
    code = main()
  File "/Users/ella/airbytehq/airbyte/airbyte-integrations/bases/source-acceptance-test/.venv/lib/python3.10/site-packages/_pytest/config/__init__.py", line 143, in main
    config = _prepareconfig(args, plugins)
  File "/Users/ella/airbytehq/airbyte/airbyte-integrations/bases/source-acceptance-test/.venv/lib/python3.10/site-packages/_pytest/config/__init__.py", line 318, in _prepareconfig
    config = pluginmanager.hook.pytest_cmdline_parse(
  File "/Users/ella/airbytehq/airbyte/airbyte-integrations/bases/source-acceptance-test/.venv/lib/python3.10/site-packages/pluggy/hooks.py", line 286, in __call__
    return self._hookexec(self, self.get_hookimpls(), kwargs)
  File "/Users/ella/airbytehq/airbyte/airbyte-integrations/bases/source-acceptance-test/.venv/lib/python3.10/site-packages/pluggy/manager.py", line 93, in _hookexec
    return self._inner_hookexec(hook, methods, kwargs)
  File "/Users/ella/airbytehq/airbyte/airbyte-integrations/bases/source-acceptance-test/.venv/lib/python3.10/site-packages/pluggy/manager.py", line 84, in <lambda>
    self._inner_hookexec = lambda hook, methods, kwargs: hook.multicall(
  File "/Users/ella/airbytehq/airbyte/airbyte-integrations/bases/source-acceptance-test/.venv/lib/python3.10/site-packages/pluggy/callers.py", line 203, in _multicall
    gen.send(outcome)
  File "/Users/ella/airbytehq/airbyte/airbyte-integrations/bases/source-acceptance-test/.venv/lib/python3.10/site-packages/_pytest/helpconfig.py", line 100, in pytest_cmdline_parse
    config = outcome.get_result()  # type: Config
  File "/Users/ella/airbytehq/airbyte/airbyte-integrations/bases/source-acceptance-test/.venv/lib/python3.10/site-packages/pluggy/callers.py", line 80, in get_result
    raise ex[1].with_traceback(ex[2])
  File "/Users/ella/airbytehq/airbyte/airbyte-integrations/bases/source-acceptance-test/.venv/lib/python3.10/site-packages/pluggy/callers.py", line 187, in _multicall
    res = hook_impl.function(*args)
  File "/Users/ella/airbytehq/airbyte/airbyte-integrations/bases/source-acceptance-test/.venv/lib/python3.10/site-packages/_pytest/config/__init__.py", line 1003, in pytest_cmdline_parse
    self.parse(args)
  File "/Users/ella/airbytehq/airbyte/airbyte-integrations/bases/source-acceptance-test/.venv/lib/python3.10/site-packages/_pytest/config/__init__.py", line 1280, in parse
    self._preparse(args, addopts=addopts)
  File "/Users/ella/airbytehq/airbyte/airbyte-integrations/bases/source-acceptance-test/.venv/lib/python3.10/site-packages/_pytest/config/__init__.py", line 1172, in _preparse
    self.pluginmanager.load_setuptools_entrypoints("pytest11")
  File "/Users/ella/airbytehq/airbyte/airbyte-integrations/bases/source-acceptance-test/.venv/lib/python3.10/site-packages/pluggy/manager.py", line 299, in load_setuptools_entrypoints
    plugin = ep.load()
  File "/opt/homebrew/Cellar/python@3.10/3.10.8/Frameworks/Python.framework/Versions/3.10/lib/python3.10/importlib/metadata/__init__.py", line 171, in load
    module = import_module(match.group('module'))
  File "/opt/homebrew/Cellar/python@3.10/3.10.8/Frameworks/Python.framework/Versions/3.10/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "/Users/ella/airbytehq/airbyte/airbyte-integrations/bases/source-acceptance-test/.venv/lib/python3.10/site-packages/_pytest/assertion/rewrite.py", line 162, in exec_module
    source_stat, co = _rewrite_test(fn, self.config)
  File "/Users/ella/airbytehq/airbyte/airbyte-integrations/bases/source-acceptance-test/.venv/lib/python3.10/site-packages/_pytest/assertion/rewrite.py", line 359, in _rewrite_test
    co = compile(tree, fn_, "exec", dont_inherit=True)
TypeError: required field "lineno" missing from alias
```
Found [this](https://airbytehq-team.slack.com/archives/C02U2SSHP9S/p1646705330115109?thread_ts=1646677517.770969&cid=C02U2SSHP9S) and confirmed minimum working version is 6.2.4 (at least for my python v 3.10.8), but pinning it this way allows us to get any `6.x` version higher, while not bumping to v7

Reply:
I don't know enough about the python ecosystem to comment on this - tagging @pedroslopez

Reply:
I'll go ahead and revert this since it's clearly not a problem in CI, we can chat about implementing it separately if it's a problem for others' local development